hack/chains/kaniko-cosign-verify.sh

```diff
@@ -29,15 +31,63 @@ title "Image digest from task result"
 kubectl get $TASKRUN_NAME -o jsonpath="{.status.taskResults[?(@.name == \"IMAGE_DIGEST\")].value}"
 echo
 
-title "Cosign verify the image"
-# Save the output data to a file so we can look at it later
-# (Actually we could just pipe it to jq because the text goes to stderr I think..?)
-show-then-run "cosign verify --key $SIG_KEY $IMAGE_URL --output-file /tmp/verify.out"
+if $INTERNAL_REGISTRY; then
+  title "Cosign verify the image - using Tekton task"
+  COSIGN_VERIFY_TASK_NAME="cosign-verify-$(openssl rand --hex 10)"
+  oc create -f <(echo "
+apiVersion: tekton.dev/v1beta1
+kind: TaskRun
+metadata:
+  name: $COSIGN_VERIFY_TASK_NAME
+spec:
+  taskRef:
+    name: cosign-verify
+    bundle: quay.io/zregvart_redhat/appstudio-tasks:2022-03-22-130413-1
+  params:
+    - name: PUBLIC_KEY
+      value: |
+$(oc get secret signing-secrets -o=go-template='{{index .data "cosign.pub"|base64decode}}' | sed 's/^/        /g')
```

Comment:
Should we specify the namespace in this command to be `tekton-chains` in case we want to create the TaskRun in another namespace?

Reply:
Or actually... why not just use $SIG_KEY here?

Reply:
> Should we specify the namespace in this command to be tekton-chains in case we want to create the TaskRun in another namespace?

I'll add it just to be on the safe side if this is copied elsewhere, the `hack/chains/kaniko-demo.sh` that is (currently) sole client of this switches to `tekton-chains`, so that's why it works now.

> Or actually... why not just use $SIG_KEY here?

I don't want to give the task access to the secret as it contains both private and public keys.